src/components/Authentication/index.js

```diff
@@ -1,29 +1,28 @@
 import React, { Component } from 'react';
 import {
-  StyleSheet, Text, View, TouchableOpacity,
+  View,
 } from 'react-native';
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-  },
-});
+// Import redux
```

Comment:
Redundant. (Make sure you fix some review that appear for both Login and Signup)